packages/graphql/src/schema/types/scalars/Date.ts

```diff
@@ -49,3 +49,5 @@ export default new GraphQLScalarType({
         return neo4j.types.Date.fromStandardDate(new Date(ast.value));
     },
 });
+
+export { DateScalar as Date };
```

Comment:
**question**: Maybe we want to use something different than `date` here? I know it is not as nice, but we should avoid name collisions with native types